CHANGES.md

```diff
@@ -24,6 +24,7 @@ Currrent Changes
   - fips, no-fips since we do not build a FIPS-validated container
   - ecdh, it was a no-op
 
+
```

Comment:
What's that extra blank line for? Did you mean to document the exec change here?